examples/log.rs

```diff
@@ -60,7 +60,7 @@ fn run(args: &Args) -> Result<(), Error> {
     } else {
         git2::SORT_NONE
     });
-    for commit in args.arg_commit.iter() {
+    for commit in &args.arg_commit {
```

Comment:
I personally prefer `for x in y.iter()` over `for x in &y`, could this commit be backed out?